pandas/core/generic.py

```diff
@@ -100,10 +101,22 @@ def __init__(self, data, axes=None, copy=False, dtype=None,
                 for i, ax in enumerate(axes):
                     data = data.reindex_axis(ax, axis=i)
 
-        object.__setattr__(self, 'is_copy', None)
+        object.__setattr__(self, '_parent', [])
```

Comment:
yes, but I may have to remove this whole list thing, couldn't get it too work properly. Its a nice idea but something is missing (e.g. lots of tests failing).